src/main/java/net/syscon/elite/repository/jpa/model/ScheduledEvent.java

```diff
@@ -0,0 +1,62 @@
+package net.syscon.elite.repository.jpa.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.JoinColumnOrFormula;
+import org.hibernate.annotations.JoinColumnsOrFormulas;
+import org.hibernate.annotations.JoinFormula;
+import org.hibernate.annotations.NotFound;
+
+import javax.persistence.*;
```

Comment:
A very meh comment but according to the GDS guidelines we shouldn't be using wildcard imports...

https://gds-way.cloudapps.digital/manuals/programming-languages/java.html#code-formatting